db/data_migration/20200924134112_update_image_display_option_for_editions.rb

```diff
@@ -0,0 +1,7 @@
+editions_with_images = Edition.find_by_sql(["SELECT e.* FROM editions e LEFT OUTER JOIN images ON e.id = images.edition_id WHERE type in ('CaseStudy', 'Consultation', 'DetailedGuide', 'FatalityNotice', 'NewsArticle', 'Publication', 'Speech') AND images.id IS NOT NULL"])
```

Comment:
This migration isn't referenced in the commit - I assume it was accidentally committed here and should be removed?

Reply:
Think it's made it's way into the wrong commit, I've moved it to the `Fix up migrations for image_display_option` where I talk about adding a data_migration